City-of-Helsinki/drupal-helfi-sote
# UHF-4063: Genesys chat with authentication (SUUNTE)

What has been done:

- block configuration for a new "Genesys chat (SUUNTE)" block
- updated cookie settings and category descriptions for the cookies required by the authentication

How to test:

1. set up SOTE instance
2. get this branch: `git checkout UHF-4063_genesys_suunte`
3. get the equivalent branch of the platform config module: `composer require drupal/helfi_platform_config:dev-UHF-4063_genesys_suunte` (see PR: https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/262)
4. run database updates: `make drush-updb`
5. import config: `make drush-cim`
6. create a content page and give it an URL alias that matches the patterns `/terveydenhoito/hammashoito` or `/terveydenhoito/hammashoito/*`
7. navigate to the page and see the chat appear with a "Tunnistaudu tästä" link in the header
8. (optional) click the link and try out the authentication - when returning to the page the link should not be visible anymore
